web/themes/custom/pineapple/sass/components/navigation/_main-menu.scss

```diff
@@ -1,2 +1,133 @@
+.menu {
+  overflow: hidden;
+  background-color: $primary;
 
+  &.responsive {
+    position: relative;
 
+    .hamburger {
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+
+    .menu-content {
+      @include breakpoint($lap) {
+        display: flex;
+        flex-direction: row;
+      }
+    }
+
+    .menu-item {
+      float: none;
+      display: block;
+    }
+
+    .expanded-container {
+      float: none;
+
+      &:hover .sub-menu {
+        display: block;
+      }
+
+      .sub-menu {
+        position: relative;
+        top: 0;
+        display: block;
+        text-indent: 10px;
+
+        @include breakpoint($lap) {
+          display: none;
+          text-indent: 0;
+        }
+      }
+
+      .menu-item {
+        display: block;
+        width: 100%;
+      }
+    }
+  }
+
+  .hamburger {
+    color: $white;
+    float: right;
+    display: block;
+    font-size: $large-font;
+
+    @include breakpoint($lap) {
+      display: none;
+    }
+  }
+
+  .menu-item {
+    float: left;
```

Comment:
Use flexbox instead of floats to do menu system.

Reply:
done